### DISCUSSION TITLE
Question about storage space used by Docker / Open-Webui Container

### DISCUSSION THREAD

**Discussion by Obsidiannite21** [open-webui/open-webui#6971] [Q&A]
Hello,

When checking the Openweb-UI docker install directory I see that its using 66gb of storage. I only have llama3.2, which is around 2gb, downloaded and running. I'm guessing that openwebui is probably less than 2gb on its own. What is the remaining 62gb for? I've attached a few screenshots that show the directories/files in question.  

There are a series of 2.5gb files that can be found in  /diff/tmp & /merged/tmp.

![Screenshot 2024-11-15 at 12 08 07 PM](https://github.com/user-attachments/assets/d83d74d3-49d3-42ef-97f0-fd5a5694c46c)
![Screenshot 2024-11-15 at 12 08 17 PM](https://github.com/user-attachments/assets/9c89a879-0f3c-4bee-8b3b-af5a10db16c1)
![Screenshot 2024-11-15 at 12 08 28 PM](https://github.com/user-attachments/assets/3397dee6-0509-4e50-9451-b930e1745c3a)

**Comment by erdnuesse** [comment]
I would pretty much like to know the same thing. Since this is just a web ui, this must contain every single increment of releases without ever cleaning up the build it seems. 
Any insight would be appreciated.

**Comment by rebelliouswhiz** [comment]
My small VM got filled up, and I found the thread here. I ended up writing a script to clean the unused images that run daily.
